app/controllers/events_controller.rb

```diff
@@ -125,4 +132,15 @@ def generate_calendar
 
     calendar
   end
+
+  def send_new_event_email(event)
+    # Send new event to all leden who have new_event_email enabled
+    if event.usergroup_id.nil?
+      unless event.attendance # If not dispuutsborrel
+        User.leden.where(new_event_mail: true).each { |user| UserMailer.mail_new_event(user, event).deliver }
```

Comment:
Willen we hier leden of aspiranten en oude lullen erbij?

Reply:
Iedereen krijgt de optie in het profiel toch? Dan zou ik gewoon iedereen doen. Voor alleen actief+aspiraten valt ook wat te zeggen. Jij mag het zeggen :)